app.py

```
@@ -38,32 +38,35 @@ def verify_origin():
+        payload, signature = token.split('.', 1)
+        expected_sig = hmac.new(
+            CHAT_API_SECRET.encode(),
+            payload.encode(),
+            hashlib.sha256
+        ).hexdigest()
+        
+        if not hmac.compare_digest(expected_sig, signature):
+            return None
+            
+        decoded_payload = base64.b64decode(payload).decode('utf-8')
+        session_id, username = decoded_payload.split('|', 1)
```

Ruff (0.15.15)

[warning] Unpacked variable `username` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)